File: WebApplication1/Models/OrderItems.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models
{
    public partial class OrderItems
    {
        public int ItemId { get; set; }
        public int? ProductId { get; set; }
        public string ProductName { get; set; }
        public int? ProductPrice { get; set; }
        public int? QuantityValue { get; set; }
        public int? OrderId { get; set; }

        public virtual Orders Order { get; set; }
    }
}

File: WebApplication1/ClientApp/src/app/model-ts/orderItems.ts
export class Orderitems implements IorderItem {

  constructor(public itemId?: number,
  public productId?: number,
  public productName?: string,
  public productPrice?: number,
  public quantityValue?: number,
  ) { }
  }


export interface IorderItem {
  itemId?: number;
  productId?: number;
  productName?: string;
  productPrice?: number;
  quantityValue?: number;
}
